sycl/doc/extensions/proposed/sycl_ext_oneapi_current_device.asciidoc

```diff
@@ -0,0 +1,140 @@
+= sycl_ext_oneapi_current_device
```

Comment:
From the description: "It also extends the new launch mechanisms with queue-less forms that utilize the default device's default queue."

Am I correct that these changes will be introduced in separate PR/spec?

Reply:
Good catch.  The PR description should be changed.

Reply:
Fixed. Took a second to realize this was the PR description and not the spec. :)